actions/novels.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+
+// eslint-disable-next-line import/prefer-default-export
+export const fetchNovels = async () => {
```

Comment:
You gave this a name and exported it with the `export const` instead of `export default` which means you need to destructure it when you import it in your utils file.

![image](https://user-images.githubusercontent.com/5775716/84534046-3ec03d00-acb7-11ea-9841-32f4d1911d6d.png)